crates/ark/src/lsp/completions/completion_item.rs

```diff
@@ -416,14 +425,14 @@ pub(super) unsafe fn completion_item_from_lazydata(
     let promise_strategy = PromiseStrategy::Simple;
 
     // Lazydata objects are never functions, so this doesn't really matter
-    let parameter_hints = ParameterHints::Enabled;
+    let parameter_hints = ParameterHints::Disabled;
 
     match completion_item_from_symbol(name, env, Some(package), promise_strategy, &parameter_hints)
     {
-        Some(item) => item,
-        None => {
+        Ok(item) => return Ok(item),
+        Err(err) => {
             // Should be impossible, but we'll be extra safe
-            bail!("Object '{name}' not defined in lazydata environment for namespace {package}")
+            bail!("Object '{name}' not defined in lazydata environment for namespace {package}: {err}")
```

Comment:
```suggestion
            Err(anyhow::anyhow!("Object '{name}' not defined in lazydata environment for namespace {package}: {err}"))
```

This should get you fallthrough for both paths (rather than `bail!` performing its early exit)